tools/linter/install/hashes/clang-tidy-linux64

```diff
@@ -0,0 +1 @@
+44103f662c866921dabd40d047788935c43402d260be10613ab2111e01726892
```

Comment:
Is this file supposed to be committed vs in the .gitignore?

Reply:
yeah it's similar to these two existing files:

https://github.com/pytorch/pytorch/blob/80cab10534e1a9ee6d2493113f169a2570979651/tools/clang_format_hash/linux64/clang-format-linux64#L1

https://github.com/pytorch/pytorch/blob/80cab10534e1a9ee6d2493113f169a2570979651/tools/clang_format_hash/mac/clang-format-mojave#L1

Reply:
Yes, we need to commit the hash since its used for verifying the binary.